src/content/configuration/dev-server.md

```diff
@@ -68,6 +68,8 @@ W> Be aware that when [exporting multiple configurations](/configuration/configu
 
 T> If you're having trouble, navigating to the `/webpack-dev-server` route will show where files are served. For example, `http://localhost:9000/webpack-dev-server`.
 
+T> If you want to recompile the bundle. e.g. after a change to the configuration. Navigating to the `/invalidate` route will instruct [webpack-dev-middleware](https://github.com/webpack/webpack-dev-middleware#invalidate) instance to recompile the bundle for you. For example, `http://localhost:9000/invalidate`.
```

Comment:
This could be shorter, clearer. Maybe:
```suggestion
T> To manually trigger a recompilation of the bundle, navigate to the `http://localhost:9000/invalidate` route (URL may differ depending on your configuration) of [webpack-dev-middleware](https://github.com/webpack/webpack-dev-middleware#invalidate).
```
Or in another way?